flocker/restapi/_schema.py

```diff
@@ -48,6 +48,11 @@ def getValidator(schema, schema_store):
     resolver = LocalRefResolver(
         base_uri=b'',
         referrer=schema, store=schema_store)
+    # XXX This breaks with the performance improvements in jsonschema 2.5.0
+    # where ``resolution_scope`` becomes a readonly property. Perhaps this
+    # isn't necessary any more?  See:
+    # https://github.com/Julian/jsonschema/pull/203
+    # FLOC-4409
     resolver.resolution_scope = b''
```

Comment:
I also think this can just be removed. In fact I'm not sure why this was added, setting base_uri to `b''` appears to have always set resolution_scope to `b''`.

I don't think their performance improvements will help us that much, but seems reasonable just to remove this line of code and update jsonschema (should let us just close FLOC-4409 immediately).

I'll leave it up to you if you want to do it now or later, but seems like it'd be less work just to remove this line, update jsonschema and re-kick the tests on this branch.